lib/src/settings.rs

```diff
@@ -258,23 +258,23 @@ impl UserSettings {
     }
 }
 
-/// Whether to report changes to the executable bit for files on unix. On
-/// windows there is no executable bit and this config is completely ignored.
+/// Whether to ignore changes to the executable bit for files on unix. On
+/// windows there is no executable bit and this config is unused.
 ///
 /// On unix we determine the default value by querying the filesystem as to
-/// whether executable bits are supported. If they are, we will report changes
-/// to files' executable bit and update the repository. Otherwise we will only
+/// whether executable bits are supported. If they are, we will respect changes
+/// to files' executable bit and update the repository. If ignored, we will only
 /// update files if they are manually changed via `jj file chmod`.
 ///
-/// On unix setting this to `false` will disable executable bit reporting and
-/// setting it to `true` will enable it regardless of our filesystem check. This
-/// can be useful for users dual-booting windows or experimenting with
+/// On unix setting this to `true` will disable automatic executable bit updates
+/// and setting it to `false` will enable it regardless of the filesystem check.
+/// This can be useful for users dual-booting windows or experimenting with
 /// nonstandard filesystems.
 ///
 /// This is not a method on UserSettings since it needs to be callable even when
 /// only having the config itself (currently only used in `Ui::with_config`).
```

Comment:
There's a lot of duplication between this text and `config.md`. I think we can keep this description short. Developers should know where they can go to read about the config option.

Reply:
Can do. I will make it reference the config documentation explicitly though.